web/core/components/issues/issue-detail-widgets/sub-issues/content.tsx

```diff
@@ -0,0 +1,172 @@
+"use client";
+import React, { FC, useCallback, useEffect, useState } from "react";
+import { observer } from "mobx-react";
+import { TIssue } from "@plane/types";
+// components
+import { DeleteIssueModal } from "@/components/issues/delete-issue-modal";
+import { CreateUpdateIssueModal } from "@/components/issues/issue-modal";
+import { IssueList } from "@/components/issues/sub-issues/issues-list";
+// hooks
+import { useIssueDetail } from "@/hooks/store";
+// helper
+import { useSubIssueOperations } from "./helper";
+
+type Props = {
+  workspaceSlug: string;
+  projectId: string;
+  parentIssueId: string;
+  disabled: boolean;
+};
+
+type TIssueCrudState = { toggle: boolean; parentIssueId: string | undefined; issue: TIssue | undefined };
+
+export const SubIssuesCollapsibleContent: FC<Props> = observer((props) => {
+  const { workspaceSlug, projectId, parentIssueId, disabled } = props;
+  // state
+  const [issueCrudState, setIssueCrudState] = useState<{
+    create: TIssueCrudState;
+    existing: TIssueCrudState;
+    update: TIssueCrudState;
+    delete: TIssueCrudState;
+  }>({
+    create: {
```

Comment:
Can we maintain the state separately?
https://react.dev/learn/choosing-the-state-structure

Reply:
A ticket has been raised for this [WEB-1861](https://app.plane.so/plane/projects/02c3e1d5-d7e2-401d-a773-45ecba45d745/issues/8caad235-0115-47ed-82ae-0112cbe1db67/). I'll take care of it.